src/components/SitewideHeader/SitewideHeader.js

```diff
@@ -126,19 +126,14 @@ const SitewideHeader = ({
 
           <Popover.Group as="nav" className="hidden md:flex space-x-10">
             <TopNavDropdownItem text="Product" subItems={features} />
+            <TopNavItem to="/backstage-comparison/" text="Roadie vs Backstage" />
             <TopNavItem to="/backstage-weekly/" text="Backstage Weekly" />
```

Comment:
I think we can temporarily replace this page to make more space in the header. At least until we manage to revive the newsletter.

```suggestion
```

Reply:
Ok, sure. I think we should keep it + revive it now, but I can open another PR for it to have that discussions.

Reply:
Meh. Up to you.